src/tqdm_publisher/_demos/_parallel_bars/_server.py

```diff
@@ -214,50 +192,41 @@ def listen_to_events():
 app = Flask(__name__)
 cors = CORS(app)
 app.config["CORS_HEADERS"] = "Content-Type"
-PORT = find_free_port()
+PORT = 3768  # find_free_port()
```

Comment:
Do you prefer this to be hardcoded? If so I'd remove the comment

Reply:
And would we even need the `find_free_port` logic is hardcoding it?

Reply:
Good point, this isn't even a function in the codebase anymore. Removed!